index.html

```diff
@@ -234,6 +232,39 @@ <h3>Skills</h3>
       </div>
     </div>
   </section>
+  <footer class="form-body">
+    <div class="card-contact">
+      <h2 class="contact-text">Contact me</h2>
+
+      <p class="contact-para">
+        If you have an application you are interested in developing, a feature
+        that you need built or a project that needs coding. I’d love to help
+        with it
+      </p>
+
+      <form action="https://formspree.io/f/mvolzpzo" method="post">
+        <ul class="form-content">
+          <li>
+            <label for="name"></label>
+            <input class="form-list" id="name" type="text" name="user_name" placeholder="First Name" maxlength="30"
+              required />
+          </li>
+          <li>
+            <label for="mail"></label>
+            <input class="form-list1" id="mail" type="email" name="user_email" placeholder="Email ID" required />
+          </li>
+          <li>
+            <label for="msg"></label>
+            <textarea class="form-list-textarea" id="msg" name="user_message" placeholder="Write your message here"
+              maxlength="500" required></textarea>
+          </li>
+          <li class="button-list">
+            <button class="button1" type="submit">Get It Touch</button>
+          </li>
+        </ul>
```

Comment:
Hello @thenarain, You have done a tremendous job aligning the form elements as required 💪🏻! However, I have a few comments to make that may lead to changes 😄.

- [ ] I have noticed that the inputs and the textarea need some padding on the left to improve on how they appear like in the figma design. Please checkout the comparison below 👇🏻 

| Figma design | Your design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/4247740/150652898-bc8c9456-e69e-49d8-84c7-7c5c196415d4.png) | ![image](https://user-images.githubusercontent.com/4247740/150652926-125d0084-b7af-4f13-aa96-b39c822c3aaa.png) |

- [ ] In the form body, I have also noticed that the button's background differs from that in the figma design. In reference to the project requirements, you are encouraged to design this portfolio form to almost look exactly like the one designed in figma. Please see the comparison below 👇🏻 

| Figma design | Your design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/4247740/150653123-19351045-f2ea-4e56-9429-63a88e810a28.png) | ![image](https://user-images.githubusercontent.com/4247740/150653090-727b8071-739a-44bb-b188-7dab9cbdc138.png) |

Reply:
Fixed both the issues.